### DISCUSSION TITLE
File uploading error in flutter

### DISCUSSION THREAD

**Discussion by Akshaykakkodi** [pocketbase/pocketbase#5092] [Q&A]
{code: validation_invalid_mime_type, message: "file_sample_100k_b_XmKKmena58.doc" mime type must be one of: image/jpeg, image/png, application/pdf, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, application/vnd.oasis.opendocument.text, application/msword, application/vnd.ms-excel, application/vnd.openxmlformats-officedocument.wordprocessingml.document.}}}, originalError: null}.

this is the error i am getting while uploading .doc file. But .docx is working fine

**Comment by ganigeorgiev** [comment]
Detecting `.doc` as `application/msword` mimetype may not be reliable because usually its identifier signature may not be at the beginning of the file which will require reading a larger portion of it, but I don't feel confident at the moment increasing the default limit (_[currently ~3072 bytes](https://github.com/gabriel-vasile/mimetype/blob/v1.4.4/mimetype.go#L15)_) since it could lead to OOM errors.

Because all doc files are essentially zip, a workaround could be to allow the upload of `application/zip`.
